assets/icons/dark/videocam.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24">

```

Comment:
> > What do you think about using this icon instead?
> > https://material.io/resources/icons/?search=camera&icon=camera_alt&style=baseline
> > i would prefer it because its simpler and feels more generic (say in the future we allow the tab to capture video stills or something like that)
> 
> One is a video camera icon and the other is a still camera icon. It doesn't seem more suitable to me since the primary function of the tab is to show a live video camera view but I don't feel strongly about it.

yeah I just feel like a "video camera" denotes recording something, while the "camera" symbolizes more of just viewing something through a lens

Reply:
Recording is coming :)

Reply:
still feel like i like the camera icon for its dual-purposeness and genericness :p

super impressed with all the work you're putting into this feature tho! Do you think we should merge in parts at a time? Say you first finish up the dashcam component, and then include enhancements in later pr. That way if someone wanted the dashcam they could start testing/using it sooner

Reply:
I will change the icon. Agreed that parts at a time is definitely the right thing to do. I am putting together an MVP.

Reply:
The intent of this pull request wasn't so much "let's merge all this in" as much as "here are some ideas to discuss".